### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by aatishpandey**
Added structure and style for Home Page
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            <input class="nav-search" type="text" placeholder="Don't Search here" name="" id="">
+
+            <div class="nav-social">
+                <div class="badge-icon-container social-button cursor-pointer">
+                    <div class="fas fa-shopping-cart badge-icon"></div>
+                    <!-- <div class="status status-digit cart">10</div> -->
+                </div>
+                <div class="badge-icon-container social-button cursor-pointer">
+                    <div class="far fa-heart badge-icon"></div>
+                    <!-- <div class="status status-digit cart">10</div> -->
```
Comment (kjais1720) [inline index.html]: remove unused comments

File: index.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                    <div class="fas fa-shopping-cart badge-icon"></div>
+                    <div>Cart</div>
+                    <!-- <div class="status status-digit cart">10</div> -->
+                </div>
+            </a>
+            <a class="link" href="/">
+                <div class="badge-icon-container">
+                    <div class="far fa-heart badge-icon"></div>
+                    <div>Favourites</div>
+                    <!-- <div class="status status-digit cart">10</div> -->
```
Comment (kjais1720) [inline index.html]: remove unused comments

**Review by kjais1720 — CHANGES_REQUESTED** [review]
Try not to use `!important` anywhere, it must be your last resort, here what you can do is increase the specificity or change the source order of the files being imported

File: index.html (hunk 1 above)
Comment (aatishpandey) [reply index.html]: fixed in recent commit

File: index.html (hunk 2 above)
Comment (aatishpandey) [reply index.html]: fixed

**Review by kjais1720 — APPROVED** [review]
All good now👍

File: css/pages/home.css [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+.nav__header__items:hover{
+    color: var(--e-primary-color);
+}
+
+.nav-search{
+    width: 35rem;
+    padding: 1rem;
+    border: none;
+    background-color: rgb(238, 238, 238);
```
Comment (tanishq20) [inline css/pages/home.css]: Use var() for colors.

File: css/pages/home.css [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+    font-size: 1.2rem;
+    outline: none;
```
Comment (tanishq20) [inline css/pages/home.css:33]: Avoid giving outline none.

File: css/pages/home.css [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+    font-size: 2rem;
+}
+
+
+.ham-links{
+    background-color: var(--white-color);
+    margin-left: auto;
+    height: 50rem;
+    padding: 0;
+    color: black;
```
Comment (tanishq20) [inline css/pages/home.css]: use var() for colors.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sahaj</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+    <link rel="stylesheet" href="https://mirageui.netlify.app/components/styles.css">
```
Comment (tanishq20) [inline index.html:10]: You can import component library CSS directly in the main CSS file.
By this, you will not have to give links in every HTML file.

**Review by tanishq20 — COMMENTED** [review]
Good use of the class name. Suggested very minor changes.

File: css/pages/home.css (hunk 3 above)
Comment (aatishpandey) [reply css/pages/home.css]: fixed

File: css/pages/home.css (hunk 4 above)
Comment (aatishpandey) [reply css/pages/home.css:33]: sure thanks for pointing out

File: css/pages/home.css (hunk 5 above)
Comment (aatishpandey) [reply css/pages/home.css]: fixed

File: index.html (hunk 6 above)
Comment (aatishpandey) [reply index.html:10]: sure thanks for pointing out

**Pull request merged**